tests/e2e/mandatory_module_test.go

```diff
@@ -137,10 +143,10 @@ var _ = Describe("Mandatory Module Installation and Deletion", Ordered, func() {
 				}).
 				Should(Succeed())
 		})
-		It("Then Kyma is in a \"Error\" State", func() {
+		It("Then Kyma is in a \"Warning\" State", func() {
 			Eventually(KymaIsInState).
 				WithContext(ctx).
-				WithArguments(kyma.GetName(), kyma.GetNamespace(), kcpClient, shared.StateError).
+				WithArguments(kyma.GetName(), kyma.GetNamespace(), kcpClient, shared.StateWarning).
```

Comment:
Where is this change coming from?

Reply:
because the new ModuleTemplates are not bound to channel, so if you try to enable a mandatory module and it has to be bound to channel in the skr cluster it will just show a warning that there are no ModuleTemplates for this channel

Reply:
Understood. Just wondering if we really want Warning for this case. But could be follow-up upon in a separate issue.

Reply:
FYI: https://github.com/kyma-project/lifecycle-manager/issues/2137